cpp/include/cudf/functions.h

```diff
@@ -90,6 +90,7 @@ gdf_error gdf_column_view(gdf_column *column, void *data, gdf_valid_type *valid,
  * @param[in] size Number of rows in the column.
  * @param[in] dtype Data type of the column.
  * @param[in] null_count The number of non-valid elements in the validity bitmask
+ * @param[in] extra_info Extra information for column description
```

Comment:
I'd add a reference to see the definition/documentation of the structure for more detail.

Reply:
(And make sure the documentation for the struct is up-to-date.)